content/scout/metrics-exporter.md

```diff
@@ -0,0 +1,167 @@
+
+## Configure Prometheus
+
+### Add a job for your organization
+
+In the Prometheus configuration file, add a new job for your organization.
+The job should include the following configuration;
+replace `ORG` with your organization name:
+
+```yaml
+scrape_configs:
+  - job_name: <ORG>
+    metrics_path: /v1/exporter/org/<ORG>
+    scheme: https
+    static_configs:
+      - targets:
+          - api.scout.docker.com
+```
+
+The address in the `targets` field is set to the domain name of the Docker Scout API, `api.scout.docker.com`.
+Make sure that there's no firewall rule in place preventing the server from communicating with this endpoint.
+
+### Scrape interval
+
+By default, Prometheus scrapes the metrics every 15 seconds.
+You can change the scrape interval by setting the `scrape_interval` field in the Prometheus configuration file at the global or job level.
+A scraping interval of 60 minutes or higher is recommended.
+
+Because of the own nature of vulnerability data, the metrics exposed through this API are unlikely to change at a high frequency.
+For this reason, the metrics endpoint has a 60-minute cache by default.
+If you set the scrape interval to less than 60 minutes, you will see the same data in the metrics for multiple scrapes during that time window.
+
+### Add bearer token authentication
+
+To scrape metrics from the Docker Scout Exporter endpoint using Prometheus, you need to configure Prometheus to use the PAT as a bearer token.
+The exporter requires the PAT to be passed in the `Authorization` header of the request.
+
+Update the Prometheus configuration file to include the `authorization` configuration block.
+This block defines the PAT as a bearer token stored in a file:
+
+```yaml
+scrape_configs:
+  - job_name: $ORG
+    authorization:
+      type: Bearer
+      credentials_file: /etc/prometheus/token
+```
+
+The content of the file should be the PAT in plain text:
+
+```console
+dckr_pat_...
+```
+
+If you are running Prometheus in a Docker container or Kubernetes pod, mount the file into the container using a volume or secret.
+
+Finally, restart Prometheus to apply the changes.
+
+## Sample project
+
+If you don't have a Prometheus server set up, you can run a [sample project](https://github.com/dockersamples/scout-metrics-exporter) using Docker Compose.
+The sample includes a Prometheus server that scrapes metrics for a Docker organization enrolled in Docker Scout,
+alongside Grafana with a pre-configured dashboard to visualize the vulnerability and policy metrics.
+
+1. Clone the starter template for bootstrapping a set of Compose services
+   for scraping and visualizing the Docker Scout metrics endpoint:
+
+   ```console
+   $ git clone git@github.com:dockersamples/scout-metrics-exporter.git
+   ```
+
+2. [Create a Docker access token](/security/for-developers/access-tokens/#create-an-access-token)
+   and store it in a plain text file named `token.txt` in the template directory.
```

Comment:
I'd rather keep the file name as `token` (without extension), otherwise we'd also need to change the `credentials_file` in the Prometheus config to include the `.txt` extension.